assets/javascripts/discourse/templates/components/chat-live-pane.hbs

```diff
@@ -9,9 +9,10 @@
         }}
       {{/if}}
 
-      {{chat-channel-title onClick=(action "onChannelTitleClick") channel=chatChannel}}
-
-      {{chat-channel-status channel=chatChannel}}
+      <div class="chat-channel-title-with-status">
+        {{chat-channel-title onClick=(action "onChannelTitleClick") channel=chatChannel}}
+        {{chat-channel-status channel=chatChannel}}
+      </div>
```

Comment:
Don't have to do it now, but usually when I see a well encapsulated `<div>` I think of a new component.

Reply:
Yeah I am adding a third component here in my other PR, we may need a todo to slice up chat-live-pane internally